compileAndBuildVLCKit.sh

```diff
@@ -547,6 +547,22 @@ buildLibVLC() {
     fingerprinter
     output_udp
     output_livehttp
+    stream_out_autodel
+    stream_out_bridge
+    stream_out_cycle
+    stream_out_delay
+    stream_out_description
+    stream_out_display
+    stream_out_dummy
+    stream_out_duplicate
+    stream_out_es
+    stream_out_gather
+    stream_out_mosaic_bridge
+    stream_out_record
+    stream_out_rtp
+    stream_out_setid
+    stream_out_smem
+    stream_out_stats
```

Comment:
I would prefer enabling all stream_out plugins.
I'm afraid we'll loose time when we will be wondering why some sout module are not working. Furthermore, these modules are very light.